pkg/cli-runtime/testing/command_table.go

```diff
@@ -122,10 +122,10 @@ type CommandTestCase struct {
 
 	// ExpectCreates asserts each resource with the resources passed to the Create method of the
 	// fake client in order.
-	ExpectCreates []Factory
+	ExpectCreates []client.Object
```

Comment:
This works, but in order to be aligned with the spirit of reconcile-runtime, all usages of the ExpectCreates inside the table should have .DeepCopy() called before comparing the value.

Reply:
I will update the test in a subsequent PR